src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/NelmioSupportPass.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
You're right. Looks like we should be safe... https://github.com/nelmio/NelmioApiDocBundle/blob/874c8752e6114fd2d384d93d48e859a680f2ae9d/DependencyInjection/LoadExtractorParsersPass.php#L36-L39